[2020-05-07 16:52:17 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:17 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:18 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:18 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:19 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:19 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:20 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:20 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:21 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:21 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:22 -0700] [9330] [ERROR] Can't connect to ('gorgonzola.cogsci.uci.edu', 20455)
2020-08-09 15:06:21,180 Scheduler started
2020-08-09 15:06:21,441 Scheduler started
2020-08-09 15:06:21,703 Scheduler started
2020-08-09 15:06:21,731 Looking for jobs to run
2020-08-09 15:06:21,743 No jobs; waiting until a job is added
2020-08-09 15:06:21,820 Looking for jobs to run
2020-08-09 15:06:21,829 No jobs; waiting until a job is added
2020-08-09 15:06:21,832 Scheduler started
2020-08-09 15:06:21,926 Scheduler started
2020-08-09 15:06:21,966 Scheduler started
2020-08-09 15:06:22,044 Looking for jobs to run
2020-08-09 15:06:22,058 No jobs; waiting until a job is added
2020-08-09 15:06:22,111 Scheduler started
2020-08-09 15:06:22,118 Scheduler started
2020-08-09 15:06:22,126 Scheduler started
2020-08-09 15:06:22,156 Looking for jobs to run
2020-08-09 15:06:22,195 No jobs; waiting until a job is added
2020-08-09 15:06:22,222 Looking for jobs to run
2020-08-09 15:06:22,232 No jobs; waiting until a job is added
2020-08-09 15:06:22,343 Looking for jobs to run
2020-08-09 15:06:22,359 No jobs; waiting until a job is added
2020-08-09 15:06:22,459 Looking for jobs to run
2020-08-09 15:06:22,463 Looking for jobs to run
2020-08-09 15:06:22,486 No jobs; waiting until a job is added
2020-08-09 15:06:22,549 No jobs; waiting until a job is added
2020-08-09 15:06:22,578 Looking for jobs to run
2020-08-09 15:06:22,588 No jobs; waiting until a job is added
[2025-04-09 13:15:34 -0700] [4824] [CRITICAL] WORKER TIMEOUT (pid:4830)
[2025-04-30 10:39:02 -0700] [34117] [CRITICAL] WORKER TIMEOUT (pid:34121)
[2025-04-30 10:39:02 -0700] [34117] [CRITICAL] WORKER TIMEOUT (pid:34128)
[2025-04-30 18:32:23 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35300)
[2025-04-30 18:32:53 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35302)
[2025-04-30 18:33:35 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35306)
[2025-04-30 18:33:47 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35301)
[2025-04-30 18:33:47 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35305)
[2025-04-30 18:33:50 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35304)
[2025-04-30 18:33:50 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35328)
[2025-04-30 18:33:50 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35329)
[2025-04-30 18:34:23 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35303)
[2025-04-30 18:34:23 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35334)
[2025-04-30 18:34:24 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35333)
[2025-04-30 18:34:25 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35330)
[2025-04-30 18:34:32 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35308)
[2025-04-30 18:34:44 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35331)
[2025-04-30 18:34:57 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35338)
[2025-04-30 18:35:02 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35335)
[2025-04-30 18:35:02 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35339)
[2025-04-30 18:35:14 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35332)
[2025-04-30 18:35:15 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35337)
[2025-04-30 18:35:19 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35341)
[2025-04-30 18:35:36 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35343)
[2025-04-30 18:35:37 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35344)
[2025-04-30 18:35:49 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35345)
[2025-04-30 18:35:53 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35340)
[2025-04-30 18:36:12 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35351)
[2025-04-30 18:36:17 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35350)
[2025-04-30 18:36:27 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35346)
[2025-04-30 18:36:28 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35347)
[2025-04-30 18:36:36 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35348)
[2025-04-30 18:36:47 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35354)
[2025-04-30 18:36:51 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35353)
[2025-04-30 18:36:52 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35352)
[2025-04-30 18:36:52 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35355)
[2025-04-30 18:37:04 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35358)
[2025-04-30 18:37:11 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35342)
[2025-04-30 18:37:22 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35307)
[2025-04-30 18:37:28 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35360)
[2025-04-30 18:37:28 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35363)
[2025-04-30 18:37:38 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35357)
[2025-04-30 18:37:53 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35359)
[2025-04-30 18:37:57 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35364)
[2025-04-30 18:38:02 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35368)
[2025-04-30 18:38:03 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35365)
[2025-04-30 18:38:14 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35361)
[2025-04-30 18:38:32 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35369)
[2025-04-30 18:38:37 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35371)
[2025-04-30 18:39:07 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35362)
[2025-04-30 18:53:11 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35367)
[2025-04-30 23:40:46 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35372)
[2025-04-30 23:40:49 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35374)
[2025-04-30 23:40:49 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35378)
[2025-04-30 23:41:04 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35366)
[2025-04-30 23:41:24 -0700] [35297] [CRITICAL] WORKER TIMEOUT (pid:35376)
2025-05-03 13:44:24,009 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 13:44:27,469 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 13:44:38,356 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 13:45:06,887 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 13:45:36 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40425)
[2025-05-03 13:46:45 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40420)
[2025-05-03 13:47:24 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40426)
[2025-05-03 13:49:40 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40421)
[2025-05-03 13:52:53 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40448)
[2025-05-03 13:53:14 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40422)
[2025-05-03 13:55:29 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40452)
[2025-05-03 14:00:24 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40423)
[2025-05-03 14:00:40 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40450)
[2025-05-03 14:01:15 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40465)
[2025-05-03 14:01:29 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40470)
[2025-05-03 14:02:50 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40461)
[2025-05-03 14:03:16 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40457)
[2025-05-03 14:03:16 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40472)
[2025-05-03 14:05:39 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40427)
[2025-05-03 14:05:50 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40479)
[2025-05-03 14:09:23 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40481)
[2025-05-03 14:09:23 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40482)
[2025-05-03 14:09:30 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40477)
[2025-05-03 14:09:58 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40419)
[2025-05-03 14:10:53 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40469)
[2025-05-03 14:10:54 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40462)
[2025-05-03 14:12:53 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40474)
[2025-05-03 14:24:04 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40497)
[2025-05-03 14:29:52 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40478)
[2025-05-03 14:32:44 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40534)
[2025-05-03 14:36:50 -0700] [40416] [CRITICAL] WORKER TIMEOUT (pid:40493)
2025-05-03 14:51:28,870 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:51:38,037 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 14:51:41 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40662)
2025-05-03 14:51:52,473 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:51:55,678 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:52:07,397 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 14:52:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40667)
[2025-05-03 14:52:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40670)
[2025-05-03 14:52:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40664)
[2025-05-03 14:52:41 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40701)
[2025-05-03 14:52:47 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40668)
2025-05-03 14:52:51,080 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:52:57,803 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 14:53:40 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40663)
[2025-05-03 14:53:40 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40705)
2025-05-03 14:54:16,601 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:54:20,202 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-03 14:54:38,830 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 14:54:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40706)
[2025-05-03 14:54:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40709)
2025-05-03 14:54:49,911 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-03 14:55:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40704)
[2025-05-03 14:55:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40712)
[2025-05-03 14:56:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40710)
[2025-05-03 14:56:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40715)
[2025-05-03 15:10:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40716)
[2025-05-03 15:11:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40665)
[2025-05-03 15:11:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40717)
[2025-05-03 15:12:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40666)
[2025-05-03 15:12:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40711)
[2025-05-03 15:13:28 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40669)
[2025-05-03 15:13:28 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40702)
[2025-05-03 15:13:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40729)
[2025-05-03 15:13:39 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40730)
[2025-05-03 15:14:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40718)
[2025-05-03 15:14:38 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40733)
[2025-05-03 15:25:01 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40703)
[2025-05-03 15:25:01 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40740)
[2025-05-03 15:34:20 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40736)
[2025-05-03 15:34:20 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40739)
[2025-05-03 15:37:31 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40731)
[2025-05-03 15:37:31 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40737)
[2025-05-03 15:37:37 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40781)
[2025-05-03 15:51:22 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40771)
[2025-05-03 15:53:24 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40738)
[2025-05-03 15:55:43 -0700] [40659] [CRITICAL] WORKER TIMEOUT (pid:40741)
